website/Makefile

```diff
@@ -65,6 +71,14 @@ $(latest_scaladoc_dir)/index.html: $(latest_version) | $(latest_scaladoc_dir)
 	  rm -rf temp.jar
 	touch $@
 
+$(snapshot_scaladoc_jar): $(chisel-non-website)
+	cd $(chisel_dir) && ./mill unipublish.docJar
+
+$(snapshot_scaladoc_dir)/index.html: $(snapshot_scaladoc_jar) | $(snapshot_scaladoc_dir)
+	rm -rf $(snapshot_scaladoc_dir)/*
+	cd $(snapshot_scaladoc_dir) && unzip -q $<
+	touch $@
```

Comment:
I thought this was unnecessary, but if the timestamp comes from the archive, then it is necessary. Nice catch.